django_rq/urls.py

```diff
@@ -1,21 +1,24 @@
-from django.conf.urls import patterns, url
```

Comment:
The old approach is deprecated (and ugly!), see https://docs.djangoproject.com/en/1.8/ref/urls/#patterns